exams/factories.py

```diff
@@ -34,6 +35,48 @@ class Meta:
         model = ExamProfile
 
 
+class ExamRunFactory(DjangoModelFactory):
+    """
+    Factory for ExamRun
+    """
+    course = SubFactory(CourseFactory)
+    exam_series_code = factory.Faker('lexify', text="????_MicroMasters")
+    date_first_schedulable = factory.LazyFunction(
+        lambda: FAKE.date_time_this_year(before_now=True, after_now=False, tzinfo=pytz.utc)
+    )
+    date_last_schedulable = factory.LazyFunction(
+        lambda: FAKE.date_time_this_year(before_now=False, after_now=True, tzinfo=pytz.utc)
+    )
+    date_first_eligible = factory.LazyFunction(
+        lambda: FAKE.date_time_this_year(before_now=False, after_now=True, tzinfo=pytz.utc).date()
+    )
+    date_last_eligible = factory.LazyAttribute(
```

Comment:
I'm not quite sure I follow how that could happen since it's based on `date_first_eligible` which is only `after_now=True`, that should be ok, correct?